steadybit/README.md

```diff
@@ -0,0 +1,66 @@
+# Steadybit
+
+## Overview
+
+[Steadybit][1] is a chaos-engineering platform allowing you to simulate turbulent conditions in a controlled way to improve system reliability and guide your organization to better incident management.
+
+Through this integration, you can leverage Datadog information within chaos-engineering experiments and see chaos-engineering activity in the form of events within Datadog.
+
+## Setup
+
+The integration between Datadog and Steadybit is done through the [Steadybit Datadog extension][2]. The extension interacts with Datadog's API to gather information about monitors and report events to Datadog.
+
+### Prerequisites
+
+You will need a [free or paid Steadybit license][3]. The integration supports Steadybit's SAAS and on-premises offering.
+
+### Installation
+
+Several [installation methods are supported][4]. The simplest way to install the Steadybit Datadog extension is through our dedicated Helm chart.
+
+To learn more about the supported values for `datadog.siteParameter` and `datadog.siteUrl`, please see [Datadog's site documentation page][5].
+
+```
+helm repo add steadybit https://steadybit.github.io/helm-charts
+helm repo update
+
+helm upgrade steadybit-extension-datadog \
+  --install \
+  --wait \
+  --timeout 5m0s \
+  --create-namespace \
+  --namespace steadybit-extension \
+  --set datadog.apiKey="{{API_KEY}}" \
+  --set datadog.applicationKey="{{APPLICATION_KEY}}" \
+  --set datadog.siteParameter="{{SITE_PARAMETER}}" \
+  --set datadog.siteUrl="{{SITE_URL}}" \
+  steadybit/steadybit-extension-datadog
+```
+
+### Validation
+
+Once the Steadybit Datadog extension is running, you can see a list of Datadog monitors within the *Landscape* tab in Steadybit.
+
+## Data Collected
+
+### Metrics
+
+Steadybit does not include any metrics.
+
+### Service Checks
+
+Steadybit does not include any service checks.
+
+### Events
+
+Steadybit reports events to Datadog indicating chaos engineering activity. All such events carry the `source:steadybit` tag.
```

Comment:
```suggestion
Steadybit reports events to Datadog indicating chaos-engineering activity. All such events carry the `source:steadybit` tag.
```

Reply:
Chaos engineering is generally not hyphenated (in literature and online media). I would therefore not accept this suggestion.

Reply:
Thanks for confirming that. I've added suggestions to remove the hyphen from the overview section.